templates/kronos.html

```diff
@@ -148,10 +167,36 @@ <h1>Sorry, no data available for that coursecode</h1>
                     zeroline: true
                 }
             };
+            var layout1 = {
+                title: 'Grade-Distributions',
```

Comment:
Can you change the title to "Grade- Distribution for {{course_code}}" or something? Also, make changes in both `layout1` and `layout`.